#
# L0_Probance_PHM_DMリスト_有効期限なし取得
#

_export:

# エラー処理
_error:
  echo>: "L0_Probance_PHM_DMリスト_有効期限なし取得でエラーが発生しました。"

# 処理開始メッセージ出力
+start_msg:
  echo>: "L0_Probance_PHM_DMリスト_有効期限なし取得 処理開始"

# データ取得
+load_l0_probance_no_xpirationdate_dm_list:
  py>: python.l1_kotsukotsu_decision_pred.main
  docker:
    image: 'digdag/digdag-python:3.9'
  _env:
    probance_path: '/home/td-user/probance/python'
    common_py_path: '/home/td-user/common/python'
    TD_API_KEY: ${secret:td.apikey}
    TD_API_SERVER: ${secret:td.endpoint}
    DEST_DATABASE: ${td.database}
    DEST_TABLE: ${l0_probance_no_xpirationdate_dm_list_table}
    HOSTNAME: ${secret:hoge} 
    PORT: ${secret:hoge} 
    USERNAME: ${secret:hoge} 
    PASSWORD: ${secret:hoge} 
    REMOTE_PATH_PREFIX: ${secret:hoge} 
    LOCAL_PATH_PREFIX: ${secret:hoge} 
    PATH_MATCH_PATTERN: ${secret:hoge} 
  table: ${l0_probance_no_xpirationdate_dm_list_table}

# 処理終了メッセージ出力
+end_msg:
  echo>: "L0_Probance_PHM_DMリスト_有効期限なし取得 処理終了"
